scalanet/src/io/iohk/scalanet/peergroup/ReqResponseProtocol.scala

```diff
@@ -175,7 +176,7 @@ object ReqResponseProtocol {
   def getUdpReqResponseProtocolClient[M](
```

Comment:
there is `case object Udp extends TransportProtocol` I would re-name it to `DynamicUDP`

Reply:
Sure thing.